.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ default_language_version:
 exclude: "{{cookiecutter.repo_name}}"
```

Comment:
We can bump precommit version also to 2.19.0
https://github.com/pre-commit/pre-commit/releases

Reply:
(also need to bump in pyproject.toml)